src/visualizers/Chaos.ts

```diff
@@ -171,41 +199,18 @@ class Chaos extends P5Visualizer(paramDesc) {
     checkParameters(params: ParamValues<typeof paramDesc>) {
         const status = super.checkParameters(params)
 
-        if (params.corners < 2) {
-            status.addError('The number of corners must be an integer > 1.')
-        }
-        if (params.frac < 0 || params.frac > 1) {
-            status.addError('The fraction must be between 0 and 1 inclusive.')
-        }
-        if (params.walkers < 1) {
+        if (params.highlightWalker >= params.walkers) {
             status.addError(
-                'The number of walkers must be a positive integer.'
+                'The highlighted walker must be less than '
+                    + 'the number of walkers.'
             )
-        }
-        if (params.gradientLength < 1) {
-            status.addError(
-                'The colour cycle length must be a positive integer.'
+            this.statusOf.highlightWalker.addWarning(
+                'must be less than the number of walkers'
```

Comment:
these two also mis-match on whether period is included... would be nice to be consistent, whichever way we go